app/controllers/image_controller.rb

```diff
@@ -121,7 +126,7 @@ def advanced_search
     show_selected_images(query)
   rescue StandardError => e
     flash_error(e.to_s) if e.present?
-    redirect_to(controller: :observations, action: :advanced_search)
+    redirect_to(search_advanced_path)
```

Comment:
This is a bug fix.
(I think it originally was
```ruby
controller: :observation, action: :advanced_search
```
Then in the ObservationController refactor, got changed via search-and-replace to:
```ruby
controller: :observations, action: :advanced_search
```
and then was overlooked because there was no relevant test assertion.